presto-hive/src/test/java/com/facebook/presto/hive/TestHivePushdownFilterQueries.java

```diff
@@ -654,6 +654,24 @@ public void testPathColumn()
         assertQuerySucceeds(session, "SELECT linenumber, \"$path\" FROM lineitem WHERE length(\"$path\") % 2 = linenumber % 2");
     }
 
+    @Test
+    public void testTextfileAndOrcTableFormatsWithPushdown()
```

Comment:
@bhhari Now, that you are testing 2 formats, the question is why not test all of them in a loop? See TestHiveIntegrationSmokeTest.

```
    @Test
    public void testCreateEmptyBucketedPartition()
    {
        for (TestingHiveStorageFormat storageFormat : getAllTestingHiveStorageFormat()) {
            testCreateEmptyBucketedPartition(storageFormat.getFormat());
        }
    }
```